pages/common/git-write-tree.md

```diff
@@ -0,0 +1,11 @@
+# git write-tree
+
+> Low level utility to create a tree object from the current index.
+
```

Comment:
This is the simplest form of the command.
```suggestion

- Create a tree object from the current index:

`git write-tree`

```

Reply:
command added.